From a3102c14941d28b474ff1ceb58ef0f542af6353a Mon Sep 17 00:00:00 2001
From: John Cox <jc@kynesim.co.uk>
Date: Wed, 4 Aug 2021 15:14:31 +0100
Subject: [PATCH 534/552] Revert "configs: Remove non-building
 CONFIG_VIDEO_RPIVID"

This reverts commit c01ce62576fe077bce8bc959f7624bad18af6dac.
---
 arch/arm/configs/bcm2711_defconfig   | 2 ++
 arch/arm64/configs/bcm2711_defconfig | 1 +
 2 files changed, 3 insertions(+)

diff --git a/arch/arm/configs/bcm2711_defconfig b/arch/arm/configs/bcm2711_defconfig
index 18253657cac0..dba7656d51d4 100644
--- a/arch/arm/configs/bcm2711_defconfig
+++ b/arch/arm/configs/bcm2711_defconfig
@@ -674,6 +674,7 @@ CONFIG_SERIAL_SC16IS7XX_SPI=y
 CONFIG_SERIAL_DEV_BUS=y
 CONFIG_TTY_PRINTK=y
 CONFIG_HW_RANDOM=y
+CONFIG_RAW_DRIVER=y
 CONFIG_TCG_TPM=m
 CONFIG_TCG_TIS_SPI=m
 CONFIG_RANDOM_TRUST_BOOTLOADER=y
@@ -1316,6 +1317,7 @@ CONFIG_R8712U=m
 CONFIG_R8188EU=m
 CONFIG_VT6656=m
 CONFIG_STAGING_MEDIA=y
+CONFIG_VIDEO_RPIVID=m
 CONFIG_FB_TFT=m
 CONFIG_FB_TFT_AGM1264K_FL=m
 CONFIG_FB_TFT_BD663474=m
diff --git a/arch/arm64/configs/bcm2711_defconfig b/arch/arm64/configs/bcm2711_defconfig
index a83b472a0085..4a8c218f8fdd 100644
--- a/arch/arm64/configs/bcm2711_defconfig
+++ b/arch/arm64/configs/bcm2711_defconfig
@@ -1321,6 +1321,7 @@ CONFIG_R8712U=m
 CONFIG_R8188EU=m
 CONFIG_VT6656=m
 CONFIG_STAGING_MEDIA=y
+CONFIG_VIDEO_RPIVID=m
 CONFIG_ASHMEM=y
 CONFIG_FB_TFT=m
 CONFIG_FB_TFT_AGM1264K_FL=m
-- 
2.33.0

